Automattic/wp-calypso
# Reader: Search on Manage connect current subs

Implementing https://github.com/Automattic/wp-calypso/issues/12953

This PR is just for listing all of the currently followed subs.
Styling will happen in a future pr.  as well as the appearance of the email subs popout.

Implementation Notes:
1. since the display logic for "list of sites" is the same whether it is from search results or subs, i moved it to a common component: _WindowScrollingSites_.

To test: 
1. Go to https://calypso.live/following/manage?branch=add/reader/current-subs
2. verify that subscriptions appear when there is no query.  After a query, subs should disappear in favor of search results
